lib/block-patterns/media-text-arquitecture.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Media and text with image on the right.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Media and text with image on the right', 'Block pattern title', 'default' ),
+	'categories'  => array( 'text' ),
```

Comment:
```suggestion
	'categories'  => array( 'header' ),
```

Our other two Media & Text patterns is in the "Header" category, so I think this one makes more sense there too.